app/views/admin/ndbn_members/index.html.erb

```diff
@@ -0,0 +1,79 @@
+<section class="content-header">
+  <div class="container-fluid">
+    <div class="row mb-2">
+      <div class="col-sm-6">
+        <% content_for :title, "Community - NDBN Member Upload" %>
+        <h1>NDBN Member Upload</h1>
+      </div>
+      <div class="col-sm-6">
+        <ol class="breadcrumb float-sm-right">
+          <li class="breadcrumb-item"><%= link_to(admin_ndbn_members_path) do %>
+              <i class="fa fa-dashboard"></i>
+              Home
+            <% end %>
+          </li>
+        </ol>
+      </div>
+    </div>
+  </div><!-- /.container-fluid -->
+</section>
+
+<section class="content">
+  <div class="container-fluid">
+    <div class="row">
+      <!-- left column -->
+      <div class="col-md-12">
+        <!-- jquery validation -->
+        <div class="card card-primary">
+          <div class="card-header">
+            <h3 class="card-title">Upload NDBN Members</h3>
+          </div>
+          <!-- /.card-header --> <!-- form start -->
+          <div class="card-body">
+            <div class="row">
+              <p>Update the current NDBN members by uploading a CSV file of the
+                current members.</p>
+            </div>
+            <%= form_with(url: admin_ndbn_members_path) do |form| %>
+              <div class="row">
+                <%= form.label :member_file, "Select CSV File to Upload" %>
+              </div>
+              <%= form.file_field :member_file, required: true %>
+              <%= submit_button({ text: "Upload" }) %>
+            <% end %>
+          </div>
+        </div>
+        <!-- /.card -->
+      </div>
+    </div>
+    <!-- /.row -->
+  </div>
+  <!-- /.container-fluid -->
+  <div class="container-fluid">
+    <div class="row">
+      <div class="col-12">
+        <!-- Default box -->
+        <div class="card">
+          <div class="card-body p-0">
+            <table class="table">
+              <thead>
+                <tr>
+                  <th>NDBN Member Number</th>
+                  <th>NDBN Member Name</th>
+                </tr>
+              </thead>
+              <tbody>
+                <%- ndbn_members.each do |member| %>
+                  <tr>
+                    <td><%= member.ndbn_member_id %></td>
+                    <td><%= member.account_name %></td>
```

Comment:
Do we need to paginate this? 

Actually, do we even need to display this data? We never have in the past. cc: @cielf

Reply:
Just in case something's not working or you aren't sure that it uploaded. It could be useful just to be able to look at all the members. 

I think this was more useful before it became a super admin only feature. But it still might have some utility, I don't necessarily think you lose anything from it?

Reply:
Every piece of code we add comes with a hidden maintenance cost. E.g. when upgrading, it's another code path we need to ensure works. If we don't have a good business / use case for it, I'd prefer leaving it out. But I'll leave it up to @cielf who is likely going to be the resident super admin to decide. :grin:

Reply:
@cielf sounds good. given we want to leave it in, do we want to paginate? I honestly feel like it might be better to not paginate just in case you want to quickly ctrl+f to search of a particular id or member.

Reply:
No need to paginate, IMO.